README.md

```diff
@@ -180,9 +187,9 @@ The following is an example of what a config file might look like:
         ]
     },
     "flag": {
-        "field": "STATUS",
```

Comment:
@aparker-cityworks STATUS and the values SUBMITTED and RECEIVED are the fields/values that are provided in the layers created by ArcGIS. Just wanted to confirm that you guys are modifying this schema to include a field 'processed' with the values Yes/No.